en/docs/install-and-setup/setup/api-controller/managing-common-api-policies/managing-common-api-policies.md

```diff
@@ -0,0 +1,114 @@
+# Managing Common API Policies
+
+## Add Common API Policies in an environment
+
+You can add Common API Policies via the Publisher Portal.
+However, **WSO2 API Controller (apictl)** allows you to create API Policies without using the Publisher Portal. For more information on adding Common API Policies, see [Importing Common API Policies Via Dev First Approach]({{base_path}}/install-and-setup/setup/api-controller/managing-apis-api-products/importing-common-api-policies-via-dev-first-approach).
+
+## Get Common API Policies in an environment
+
+Follow the instructions below to display a list of Common API Policies in an environment using apictl:
+
+1.  Make sure that the WSO2 API Manager (WSO2 API-M) 4.1.0 version is started and that the 4.1.0 version of apictl is set up.   
```

Comment:
Let's avoid the version number. They will see this in the 4.1.0 docs so the version should be obvious to most users plus it will help when we cut newer branches in future.